lochness/xnat/__init__.py

```diff
@@ -50,7 +50,8 @@ def sync(Lochness, subject, dry=False):
                     yaxil.download(auth, experiment.label,
                                    project=experiment.project,
                                    scan_ids=['ALL'], out_dir=tmpdir,
-                                   in_mem=False, attempts=3)
+                                   in_mem=False, attempts=3,
+                                   out_format='native')
```

Comment:
Can we get a comment about the necessity of `out_format`?

Reply:
out_format=`flat` was the default option for the `yaxil.download` and this led to a single directory having all the dicom files regardless of different series. `out_format='native'` is set in order to have separate directories for different series.